### PR TITLE
enhance: Enhance import integration tests and logs

### PR DESCRIPTION
**Pull request by bigsheeper**
1. Optimize the import process: skip subsequent steps and mark the task as complete if the number of imported rows is 0.
2. Improve import integration tests:
 a. Add a test to verify that autoIDs are not duplicated
 b. Add a test for the corner case where all data is deleted
 c. Shorten test execution time
3. Enhance import logging:
 a. Print imported segment information upon completion
 b. Include file name in failure logs

issue: https://github.com/milvus-io/milvus/issues/42488, https://github.com/milvus-io/milvus/issues/42518

### CODE REVIEW

File: tests/integration/import/auto_id_test.go [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+	for i := 0; i < fileNum; i++ {
+		wg.Add(1)
+		i := i
+		go func() {
+			defer wg.Done()
+			importResp, err := c.Proxy.ImportV2(ctx, importReqs[i])
+			s.NoError(err)
+			s.Equal(int32(0), importResp.GetStatus().GetCode())
+			log.Info("Import result", zap.Any("importResp", importResp))
+			time.Sleep(2 * time.Second)
```
Comment (bigsheeper) [inline tests/integration/import/auto_id_test.go]: /hold
remove sleep after pr https://github.com/milvus-io/milvus/pull/42415 merged

Comment (bigsheeper) [reply tests/integration/import/auto_id_test.go]: /unhold
done

**Review by czs007 — APPROVED** [review]
/lgtm
